libs/network/socketserver.c

```diff
@@ -487,7 +485,7 @@ bool socketClientConnectionsIdle(JsNetwork *net) {
         }
         // Now read data if possible (and we have space for it)
         if (!receiveData || !hadHeaders) {
-          int num = netRecv(net, sckt, buf, net->chunkSize);
+          int num = netRecv(net, sckt, buf, sizeof(buf));
```

Comment:
I think this is a merge error? It should definitely be chunkSize now, as `buf` is a `char*`